content/docs/2.10/concepts/admission-webhooks.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Admission Webhook"
+description = "Automatically validate resource changes to prevent misconfiguration and enforce best practices"
+weight = 100
```

Comment:
Let's make sure it gets pushed down the list:
![image](https://user-images.githubusercontent.com/4345663/210947878-1127dc74-6171-42d2-b9bc-178199356114.png)

Reply:
Yeah, I didn't know what the `weight` means, but now I got it! I move it between External Scalers and Troubleshooting